src/resources/Registrations/generalPartnership.ts

```diff
@@ -1,15 +1,24 @@
-import { IncorporationResourceIF } from '@/interfaces'
+import { RegistrationResourceIF } from '@/interfaces'
 import { CorpTypeCd, FilingCodes, NameRequestTypes } from '@/enums'
 import { GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module'
 import { RegistrationSteps } from '@/resources/Registrations/stepTemplates'
 
-export const GeneralPartnershipResource: any = {
+export const GeneralPartnershipResource: RegistrationResourceIF = {
   entityType: CorpTypeCd.PARTNERSHIP,
   displayName: GetCorpFullDescription(CorpTypeCd.PARTNERSHIP),
   nameRequestType: NameRequestTypes.GP,
   steps: RegistrationSteps,
   filingData: [{
     entityType: CorpTypeCd.PARTNERSHIP,
     filingTypeCode: FilingCodes.REGISTRATION_GP
-  }]
+  }],
+  reviewAndConfirm: {
+    completingPartyStatement: {
+      certifyStatementHeader: null,
+      certifyStatements: [],
+      certifyClause: `Note: It is an offence to make or assist in making a false or misleading statement in a record
+      filed under the Partnership Act. A person who commits this offence is subject to a maximum fine of $5,000.`,
```

Comment:
I know that it's legal to have multi-line strings when you use the back-tick, and I know it's uglier to concatenate strings, but what you get here is a bunch of whitespace between words on different lines. HTML usually displays these as a single space, but somehow it still bugs me. I recommend you concat strings instead.

Reply:
For all entities in all resources? 
If using back-ticks causes issues i'm all for but i'm not sure i want to update every instance of back-tick string templating unless there is a good reason too.

Reply:
How about just updating this new code?

Also, nth lines should be indented.

Reply:
Updated